io_ogre/config.py

```diff
@@ -116,7 +117,8 @@ def load_config():
             with open( CONFIG_FILEPATH, 'rb' ) as f:
                 config_dict = pickle.load( f )
         except:
-            print('[ERROR]: Can not read config from %s' %CONFIG_FILEPATH)
+            #print('[ERROR]: Can not read config from %s' %CONFIG_FILEPATH)
+            logger.error(' Can not read config from %s' % CONFIG_FILEPATH)
```

Comment:
```suggestion
            logger.error(' Can not read config from %s', CONFIG_FILEPATH)
```
prefer this form as it avoids string expansion when that specific log level is disabled.

This is rather for future PRs - I would merge this regradless of this change.

Reply:
using , instead of % defers the string expansion until the actual logging happens. This is a minor optimization and just nitpicking on my side..